js/main.js

```diff
@@ -0,0 +1,221 @@
+    xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + data.currentFour[0]);
+    xhr.onreadystatechange = function () {
+      if (this.readyState == 4) {
+        if (this.status == 200) {
+          resolve(this.responseText);
+        } else {
+          reject('Call 1 Failed');
+        }
+      }
+    };
+    xhr.send();
+  })
+}
+
+function getPokemonName_2() {
+  return new Promise(function (resolve, reject) {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function () {
+      if (this.readyState == 4) {
+        if (this.status == 200) {
+          resolve(this.responseText)
+        } else {
+          reject('Call 2 Failed');
+        }
+      }
+    };
+    xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + data.currentFour[1]);
+    xhr.send();
+  })
+}
+
+function getPokemonName_3() {
+  return new Promise(function (resolve, reject) {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function () {
+      if (this.readyState == 4) {
+        if (this.status == 200) {
+          resolve(this.responseText)
+        } else {
+          reject('Call 3 Failed');
+        }
+      }
+    };
+    xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + data.currentFour[2]);
+    xhr.send();
+  })
+}
+
+function getPokemonName_4() {
+  return new Promise(function (resolve, reject) {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function () {
+      if (this.readyState == 4) {
+        if (this.status == 200) {
+          resolve(this.responseText)
+        } else {
+          reject('Call 4 Failed');
+        }
+      }
+    };
+    xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + data.currentFour[3]);
+    xhr.send();
+  })
+}
+
+function createQuizQuestion() {
+  Promise.all([getPokemonName_1(), getPokemonName_2(), getPokemonName_3(), getPokemonName_4()]).then(function (values) {
+    //Appending random Image to body
+    var randomInteger = Math.floor(Math.random() * 4)
+    console.log(shuffledFour[randomInteger])
+    var $quizDiv = document.querySelector('#quiz');
+    var $quizContainer = document.createElement('div');
+    $quizContainer.className = "quizContainer row justify-center"
+    $quizDiv.appendChild($quizContainer);
+    var $img = document.createElement('img');
+    $img.setAttribute('src', JSON.parse(values[randomInteger]).sprites.other['official-artwork'].front_default)
+    $quizContainer.appendChild($img);
+    $img.addEventListener('load', function () {
+      //BAR LOAD 
+      var $barRow = document.createElement('div');
+      $barRow.className = "bar";
+      $quizContainer.prepend($barRow);
+      var $inRow = document.createElement('div');
+      $inRow.className = "in";
+      $barRow.appendChild($inRow);
+      //QUERYING FOR DOTS
+      var $dots = document.querySelectorAll('.col-tenth')
+      console.log($dots);
+      //SET TIMEOUT, 
+      var timeBar = setTimeout(myTimer, 10000)
+
+      function myTimer() {
```

Comment:
_**YUCK**_

Please try to avoid the pattern myWhatever. "My" tells me nothing about this code's purpose. Try to choose a name that describes what this is for.